starboard/shared/libevent/socket_waiter_add.cc

```diff
@@ -45,3 +46,35 @@ bool SbSocketWaiterAdd(SbSocketWaiter waiter,
 
   return waiter->Add(socket, context, callback, interests, persistent);
 }
+
+#if SB_API_VERSION >= 16
+
+bool SbPosixSocketWaiterAdd(SbSocketWaiter waiter,
```

Comment:
Are we not going to deprecate SbSocketWaiter and SbPosixSocketWaiterCallback for SB 16?

Reply:
This change intents to add the POSIX APIs. 
The deprecation will be applied by future CL.